manifests/4.8/image-references

```diff
@@ -0,0 +1,13 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-kube-descheduler-operator
```

Comment:
copy paste from descheduler... we'll need to see if this file is necessary with CPaaS